lib/collection/variable-scope.js

```diff
@@ -131,6 +132,16 @@ _.assign(VariableScope.prototype, /** @lends VariableScope.prototype */ {
     },
 
     /**
+     * Converts a list of Variables into an object where key is `_postman_propertyIndexKey` and value is determined
+     * by the `valueOf` function
+     *
+     * @return {Object}
+     */
+    toObject: function (excludeDisabled, caseSensitive) {
```

Comment:
Good that we kept the parameters `excludeDisabled, caseSensitive` private. No need to document these though.